tests/src/test/scala/cats/tests/XorTests.scala

```diff
@@ -46,6 +47,12 @@ class XorTests extends CatsSuite {
     checkAll("Eq[ListWrapper[String] Xor ListWrapper[Int]]", SerializableTests.serializable(Eq[ListWrapper[String] Xor ListWrapper[Int]]))
   }
 
+  {
+    implicit val L = ListWrapper.semigroup[String]
```

Comment:
Yeah, the implicit scope rules are incredibly complicated (which admittedly we use as a "feature" quite a bit in cats to aid in inference). In this case we are using the curly braces to have it scoped locally to that block instead of to the whole class/object.

😈